src/messages-alert-container.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export class MessagesAlertContainer extends React.Component {
+
+  static propTypes = {
+    mailCount: React.PropTypes.number.isRequired,
+    settingsOnClick: React.PropTypes.func,
+    allMessagesOnClick: React.PropTypes.func,
```

Comment:
- [ ] Since this function isn't required, if we don't have it set in `this.props`, we should hide the `See all messages` link. 

Same with mark all as read and settings. 

That way our users can naturally hide those UI elements if the functionality isn't built out.